### PR TITLE
Fix typos that prevent formating of h5 titles

### PR DESCRIPTION
**Pull request by zeroheure**
really simple

### CODE REVIEW

File: include/command-line-processing.php [hunk 1]
```diff
@@ -70,7 +70,7 @@
 </ol>
 
 <p>You can find a detailed explanation of each of the constituent parts of the command-line in the sections that follow.</p>
-
+'
```
Comment (dlemstra) [inline include/command-line-processing.php:73]: Thanks for the h5 fixes but why was this added?

Comment (zeroheure) [reply include/command-line-processing.php:73]: probably a typo from me :-)

**Comment by dlemstra** [comment]
Closing this pull request because we changed `h5` to `h4`.
